CHANGES.rst

```diff
@@ -1,3 +1,7 @@
+0.1.13 (2024-05-23)
```

Comment:
I don't want to promise a release today (not 100% sure I even have privileges to do that), so let's change this to:

```suggestion
development (unreleased)
```